shared/common-adapters/user-proofs.desktop.js

```diff
@@ -134,8 +134,18 @@ export default class ProofsRender extends Component {
     const {loading, onClickProofMenu, showingMenuIndex} = this.props
     const pad = idx => idx > 0 ? {marginTop: globalMargins.tiny} : {}
     const missingProofsRealCSS = `
-      .user-proof-row .user-proof-row__name { text-underline: none; font-weight: normal; }
-      .user-proof-row:hover .user-proof-row__name { text-decoration: underline; font-weight: bold; }
+      .user-proof-row .user-proof-row__name {
+        text-underline: none;
+      }
+      .user-proof-row:hover .user-proof-row__name {
+        text-decoration: underline;
+      }
+      .user-proof-row .user-proof-row__name, .user-proof-row .user-proof-row__icon {
+        color: ${globalColors.black_20} !important; /* Must use important because Text has a default color which is set inline */
+      }
+      .user-proof-row:hover .user-proof-row__name, .user-proof-row:hover .user-proof-row__icon {
+        color: ${globalColors.black_60} !important;
```

Comment:
I don't think you do if you use inherit on the icon

On Friday, August 5, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/common-adapters/user-proofs.desktop.js
> <https://github.com/keybase/client/pull/3757#discussion_r73764662>:
>
> > @@ -134,8 +134,18 @@ export default class ProofsRender extends Component {
> >      const {loading, onClickProofMenu, showingMenuIndex} = this.props
> >      const pad = idx => idx > 0 ? {marginTop: globalMargins.tiny} : {}
> >      const missingProofsRealCSS = `
> > -      .user-proof-row .user-proof-row__name { text-underline: none; font-weight: normal; }
> > -      .user-proof-row:hover .user-proof-row__name { text-decoration: underline; font-weight: bold; }
> > +      .user-proof-row .user-proof-row__name {
> > +        text-underline: none;
> > +      }
> > +      .user-proof-row:hover .user-proof-row__name {
> > +        text-decoration: underline;
> > +      }
> > +      .user-proof-row .user-proof-row__name, .user-proof-row .user-proof-row__icon {
> > +        color: ${globalColors.black_20} !important; /* Must use important because Text has a default color which is set inline */
> > +      }
> > +      .user-proof-row:hover .user-proof-row__name, .user-proof-row:hover .user-proof-row__icon {
> > +        color: ${globalColors.black_60} !important;
>
> do you need this !important?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3757/files/ce75e8e19bdd53aff94a577bf1eb3f172108b6e5#r73764662>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhU1cMcwzmBBIVI4WBuz8CV0fEo5IFks5qc7AdgaJpZM4JdSQz>
> .
>